instrumentation/aws-lambda/aws-lambda-events-2.2/library/src/main/java/io/opentelemetry/instrumentation/awslambdaevents/v2_2/TracingRequestApiGatewayWrapper.java

```diff
@@ -46,6 +46,7 @@ static <T> T map(APIGatewayProxyRequestEvent event, Class<T> clazz) {
   @Override
   protected APIGatewayProxyResponseEvent doHandleRequest(
       APIGatewayProxyRequestEvent input, Context context) {
+    System.out.println("TracingRequestApiGatewayWrapper");
```

Comment:
Same `println` other place :)

Reply:
I need (to keep) them because logging works bad, but I didn't intend to merge them to the main branch